geomstats/geometry/manifold.py

```diff
@@ -82,6 +88,30 @@ def equip_with_metric(self, Metric=None, **metric_kwargs):
 
         self.metric = Metric(self, **metric_kwargs)
 
+    def equip_with_group_action(self, group_action):
+        self.group_action = group_action
+
+    def _check_equip_with_quotient_structure(self):
+        for structure_name in ("metric", "group_action"):
+            if not hasattr(self, structure_name):
+                raise ValueError(f"Need to equip with `{structure_name}` first")
+
+    def equip_with_quotient_structure(self):
+        self._check_equip_with_quotient_structure()
+
+        key = type(self.metric), self.group_action
+
+        out = self._quotient_map.get(key, None)
+        if out is None:
+            raise ValueError(f"No mapping for key: {key}")
+        FiberBundle_, Metric = out
+
+        self.fiber_bundle = FiberBundle_(space=self)
+        self.quotient_metric = Metric(space=self)
```

Comment:
Could we call this `QuotientMetric`? It could be a bit confusing otherwise.

Reply:
sure. `QuotientMetric_` instead? just for same reasoning as above.

Reply:
Yes perfect. `QuotientMetric_`